src/dxtbx/format/FormatCBFMultiTileHierarchy.py

```diff
@@ -281,21 +281,27 @@ def _detector(self):
             p.set_local_frame(fast, slow, origin)
 
             try:
-                overload = cbf.get_overload(panel_number)
+                max_trusted_value = cbf.get_overload(panel_number) - 1
                 cbf.find_category(b"array_intensities")
                 cbf.find_column(b"array_id")
                 if has_sections:
                     cbf.find_row(array_ids_detectororder[panel_number])
                 else:
                     cbf.find_row(panel_name)
-                cbf.find_column(b"undefined_value")
-                # undefined_value, interpreted as 1 less than the minimum acceptable value
-                underload = cbf.get_doublevalue()
-                trusted_range = (underload + 1, overload)
+                try:
+                    cbf.find_column(b"underload")
+                    offset = 0
+                except Exception:
+                    # By convention, if underload is not set, then assume the minimum
+                    # trusted pixel is 1 more than the undefined pixel
+                    cbf.find_column(b"undefined_value")
+                    offset = 1
+                min_trusted_value = cbf.get_doublevalue() + offset
```

Comment:
Just want to make sure. This will raise on `find_column` if the column doesn't exist, right? If the exception is on `get_doublevalue()` then that needs to be inside the `try`. @phyy-nx thoughts?

Reply:
find_column is the thing that will fail.  get_doublevalue should always succeed if find_column does, otherwise that's an error that should probably be raised.

Reply:
ok then this is the right way. thanks!